test/plugin/scenarios/jedis-scenario/bin/startup.sh

```diff
@@ -18,5 +18,5 @@
 
 home="$(cd "$(dirname $0)"; pwd)"
 
-java -jar ${agent_opts} -Dredis.host=${REDIS_HOST} -Dredis.port=${REDIS_PORT} \
+java -jar ${agent_opts} -Dredis.host=${REDIS_HOST} -Dredis.port=${REDIS_PORT} -Dskywalking.plugin.jedis.trace_redis_parameters=true\
```

Comment:
As you activate this, please update the expected segment to verify this new tag.

Reply:
Both plugins support collecting redis parameters previously, what I do was to add a switch to disable the collection. So the tag don't need to change.